proto/vtgate.proto

```diff
@@ -159,6 +159,8 @@ message Session {
 
   // MigrationContext
   string migration_context = 27;
+
+  bool tx_error_block_next_queries = 28;
```

Comment:
nit: let's make it clear, options:
1. `freeze_until_rollback`
2. `fail_until_rollback`
3. `block_until_rollback`
4. `rollback_expected`
5. `lock_until_rollback`
6. `rollback_required`
7. `reject_until_rollback`
8. `error_until_rollback`

Reply:
I changed it via 812df8f0dd6, I picked `error_until_rollback`.